### ISSUE TITLE
roadmapping new dat-ecosystem website

### ISSUE THREAD

**Issue by Hornet004** [playproject-io/contracting#13]
## `@todo`
- [ ] make new dat-ecosystem website
  - [ ] #14
  - [ ] #15 
  - [ ] #16 
  - `@output` ❓ `new dat-ecosystem website`

---

## `@info`
This directory contains the project road map for the dat-ecosystem website.

**Comment by serapath** [comment]
**feedback `2022.04.27`**

thanks. It already looks better.
The 3 linked issues are themselves todos though, so i would recommend to change this issue like this:
```
## `@todo`
- [ ] make new dat-ecosystem website
  - [ ] #14
  - [ ] #15 
  - [ ] #16 
  - `@output` ❓ `new dat-ecosystem website`

---

## `@info`
This directory contains the project road map for the dat-ecosystem website.
```

**edit:**

haha, you already improved it while i was writing feedback by making those sub issues also todos.
Now the thing is, still, the output should be the website itself i guess and the sub issue todos are steps that happen along the way, before the final output :-)

**Comment by Hornet004** [comment]
**worklog `2022.04.27`**

done

**Comment by serapath** [comment]
**feedback `2022.04.27`**

we don't have or make issues or todos to make issues or todos (which is what the roadmapping plan is)

so the trial phase itself and the practice of making todos/issues and linking outputs and inputs and always following the recipe on a daily basis that i linked [here](https://github.com/playproject-io/contracting/issues/3#issuecomment-1108143134) under **CHECKLIST:** https://hackmd.io/OzSDw7a6ShS78ZS6ITnZuA?view  ...is good enough to make the plan.

Without making a plan to make a plan we can definitely create one issue that says:
* [ ] make website
  * `@output` :question: `the website`

...and that is definitely too little, but from there on we just start refining and refining and refining using whats written in the checklist :-) ...to slowly improve the plan throughout the time we work on it - so it doesnt need really a todo or issue to make the todos or issues themselves

---

I edited it, to indent the todos, because to me they seem "sub todos" that all need to be fixed in order to produce the output of `"make new dat-ecosystem website"`

also renamed it from "making a plan" into "making the website" (that is the goal)

**Comment by Hornet004** [comment]
**worklog `2022.04.27`**

**Comment by serapath** [comment]
merged into #49
